auth-server/src/services/auth.ts

```diff
@@ -368,6 +373,10 @@ export default class AuthService {
 			throw new ForbiddenError(messages.USERNAME_ALREADY_EXISTS);
 		}
 
+		if (nameBlacklist.includes(username.toLowerCase())) {
```

Comment:
Shouldn't it also be checked vice versa?

i.e. sth like

```
for item from items... {
username.toLowerCase().includes(nameBlackList[item])
}
```

}

Reply:
https://github.com/paritytech/polkassembly/pull/1123